docs/en/docs/tutorial/extra-models.md

```diff
@@ -162,6 +162,9 @@ It will be defined in OpenAPI with `anyOf`.
 
 To do that, use the standard Python type hint <a href="https://docs.python.org/3/library/typing.html#typing.Union" class="external-link" target="_blank">`typing.Union`</a>:
 
+!!! note
+    When defining a <a href="https://pydantic-docs.helpmanual.io/usage/types/#unions" class="external-link" target="_blank">`Union`</a>, include the most specific type first, followed by the less specific type. 
```

Comment:
Good catch. I don't know if it's 100% clear without an example, do you think the same example can/need to be added here, or the note itself is already clear enough?

Reply:
I added clarification on how this behaviour can be seen in the example already within the Union section.

Reply:
@kbanc Thank you! :)